app/gui2/src/util/shortcuts.ts

```diff
@@ -317,13 +322,6 @@ function decomposeKeybindString(string: string): ModifierStringDecomposition {
 
 function parseKeybindString(string: string): Keybind | Mousebind {
   const decomposed = decomposeKeybindString(string)
-  const normalized =
-    decomposed.modifiers.length === 0
-      ? decomposed.key
-      : `${decomposed.modifiers.join('+')}+${decomposed.key}`
-  if (normalized !== string) {
-    console.warn(`Modifier string '${string}' should be '${normalized}'`)
-  }
   if (isPointer(decomposed.key)) {
```

Comment:
well, i think it's not *vital* but i think it'd be nice to sort of have builtin linting - it's true that this isn't useful for builtin shortcuts, but i think it'd be nice for user-defined shortcuts. that said, `console.warn` isn't very visible, but i don't think we have a well-defined place to put status notifications right now

Reply:
This whole lint was incorrect once you used any overrides in `normalizedKeyboardSegmentLookup`. E.g. using `OsDelete` or `Space` (which I added, since it was impossible to use `' '` due to trim in decomposition logic). After all, `decomposed.key` is simply not necessarily what you actually want to write in your binding.

Reply:
ah yeah, good point. i think this was written before the overrides and i kinda just forgot that that would invalidate this